raiden/storage/utils.py

```diff
@@ -59,4 +77,6 @@ def __getattr__(self, item):
     DB_CREATE_SNAPSHOT,
     DB_CREATE_STATE_EVENTS,
     DB_CREATE_RUNS,
+    DB_CREATE_M_USER_IDS,
```

Comment:
Can you change this to `DB_CREATE_MATRIX_USER_IDS`? same applies to the one below this as well.